stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -164,9 +184,121 @@ public ClientIOTaskResult runLocal() throws Exception {
     ClientIOTaskResult.ThreadCountResult result = context.getResult();
     LOG.info(String.format("thread count: %d, errors: %d, IO throughput (MB/s): %f", numThreads,
         result.getErrors().size(), result.getIOMBps()));
+
     return result;
   }
 
+  /**
+   * Read the log file from java agent log file.
+   *
+   * @param clientIOTaskResult client io task result
+   * @return ClientIOTaskResult with java agent info
+   * @throws IOException
+   */
+  @SuppressFBWarnings(value = "DMI_HARDCODED_ABSOLUTE_FILENAME")
+  public synchronized ClientIOTaskResult addAdditionalResult(ClientIOTaskResult clientIOTaskResult)
```

Comment:
This seems to share a lot with : https://github.com/Alluxio/alluxio/blob/master/stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java#L225

Can we refactor this out to be shared?

@bradyoo

Reply:
You don't really need to have the filter here because you can filter later in the GraphGenerator in ClientIO.